.github/workflows/actions.yml

```diff
@@ -6,14 +6,44 @@ on:
     - master
 
 jobs:
-  main:
+  basic:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - task: Lint
+            allow_failures: false
+            command: cd packages/basic && yarn install && yarn lint
+          - task: Test
+            allow_failures: false
+            command: cd packages/basic && yarn install && yarn test
     steps:
     - uses: zendesk/checkout@v2
     - name: Use Node.js
       uses: zendesk/setup-node@v2.1.2
       with:
         node-version: '12.x'
-    - run: yarn install
-    - run: yarn lint
-    - run: yarn test
```

Comment:
Since you have added lerna, you just use lerna to run these scripts instead of manually cd'ing into package dirs
see
https://github.com/lerna/lerna/tree/main/commands/run#lernarun

We do the same in zcli

Reply:
Hopefully this commit does the trick! https://github.com/zendesk/app_scaffolds/pull/2/commits/de83f721933f55276e40437d7dc0a1be5eb30f26